### PR TITLE
Kamiooka Upgrade 🍱

### PR DESCRIPTION
**Pull request by milesmanley**
Kamiooka Upgrade 🍱 - Doc clean up, version bump, Copyright update and change log WIP - Will add in the next couple of hours. 

- Bitcore / Insight implementation needs testing/checking before merging

### CODE REVIEW

File: src/chainparamsbase.cpp [hunk 1]
```diff
@@ -23,15 +23,15 @@ class CBaseMainParams : public CBaseChainParams
...
+ * Kamiooka-testnet
  */
 class CBaseTestNetParams : public CBaseChainParams
 {
 public:
     CBaseTestNetParams()
     {
         nRPCPort = 26124;
-        strDataDir = "testnet6";
+        strDataDir = "kamiooka-testnet";
```
Comment (TheTrunk) [inline src/chainparamsbase.cpp]: testnet-kamiooka is more convenient if we dont want to follow up with numbers

Comment (milesmanley) [reply src/chainparamsbase.cpp]: > The 2.0.5 merge of Zcash breaks insight explorers which are an essential piece and as of changes consensus changes it is not possible for our explorers to stay on current version. Thus bitcore shall be ported entirely prior merging this PR.
> Zcash is currently working on merging bitcore into their master.
> [zcash/zcash#3708](https://github.com/zcash/zcash/issues/3708)
> [zcash/zcash#3972](https://github.com/zcash/zcash/pull/3972)

Sounds good, I know Renuzit managed to patch Bitcore for BTCZ hope you can take a look at it over the next couple of days.

Comment (milesmanley) [reply src/chainparamsbase.cpp]: > testnet-kamiooka is more convenient if we dont want to follow up with numbers

I agree.

Comment (TheTrunk) [reply src/chainparamsbase.cpp]: > > The 2.0.5 merge of Zcash breaks insight explorers which are an essential piece and as of changes consensus changes it is not possible for our explorers to stay on current version. Thus bitcore shall be ported entirely prior merging this PR.
> > Zcash is currently working on merging bitcore into their master.
> > [zcash/zcash#3708](https://github.com/zcash/zcash/issues/3708)
> > [zcash/zcash#3972](https://github.com/zcash/zcash/pull/3972)
> 
> Sounds good, I know Renuzit managed to patch Bitcore for BTCZ hope you can take a look at it over the next couple of days.

I think Renuzit used 2.0.4 base for their insight explorer and ported the consensus changes to that 2.0.4 stream. That is also an option for us to do but I would rather resolve it in 2.0.5

Comment (milesmanley) [reply src/chainparamsbase.cpp]: > > > The 2.0.5 merge of Zcash breaks insight explorers which are an essential piece and as of changes consensus changes it is not possible for our explorers to stay on current version. Thus bitcore shall be ported entirely prior merging this PR.
> > > Zcash is currently working on merging bitcore into their master.
> > > [zcash/zcash#3708](https://github.com/zcash/zcash/issues/3708)
> > > [zcash/zcash#3972](https://github.com/zcash/zcash/pull/3972)
> > 
> > 
> > Sounds good, I know Renuzit managed to patch Bitcore for BTCZ hope you can take a look at it over the next couple of days.
> 
> I think Renuzit used 2.0.4 base for their insight explorer and ported the consensus changes to that 2.0.4 stream. That is also an option for us to do but I would rather resolve it in 2.0.5

Ah ok, made change for testnet-kamiooka directory.

**Pull request merged**